### PR TITLE
Changes to log levels for some lines

### PR DESCRIPTION
**Pull request by L0Lmaker**
*Issue #, if available:*
https://bitquill.atlassian.net/browse/AN-1040

*Description of changes:*
Changes to log levels for some lines

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: gremlin-go/driver/driverRemoteConnection.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func NewDriverRemoteConnection(
 
 	logHandler := newLogHandler(settings.Logger, settings.LogVerbosity, settings.Language)
 	if settings.Session != "" {
-		logHandler.log(Info, sessionDetected)
+		logHandler.log(Debug, sessionDetected)
```
Comment (lyndonbauto) [inline gremlin-go/driver/driverRemoteConnection.go:90]: The Client seems to have the same printout, do we want that?

**Comment by lyndonbauto** [comment]
I think some files are missing. Take a look at the protocol, I think it was recently updated but not sure if it is Info still. Only Infos should be Connect and Disconnect. Others should be error/debug

**Comment by L0Lmaker** [comment]
> I think some files are missing. Take a look at the protocol, I think it was recently updated but not sure if it is Info still. Only Infos should be Connect and Disconnect. Others should be error/debug

some redundant Protocol logs were removed, and the ones that remain were updated to Error or Debug. The only usages now of Info with the new changes are connecting to and disconnecting from `Connections, Clients, Sessions and DRC`

File: gremlin-go/driver/driverRemoteConnection.go (hunk 1 above)
Comment (L0Lmaker) [reply gremlin-go/driver/driverRemoteConnection.go:90]: The section from 89-92 is replicated in `client.go`. `NewDRC()` doesn't depend on `NewClient()` and operates independently as it generates its own Client object directly. We could utilize the `NewClient` function in `NewDRC` and get rid of lines 89-92, but I'm wondering if there was a reason it was intentionally made that way.

**Review by lyndonbauto — APPROVED** [review]

**Review by vkagamlyk — APPROVED** [review]

**Pull request merged**